src/features/defi/providers/univ2/components/UniV2Manager/Deposit/DepositContext.ts

```diff
@@ -0,0 +1,10 @@
+import { createContext } from 'react'
+
+import type { UniV2DepositActions, UniV2DepositState } from './DepositCommon'
+
+interface IDepositContext {
+  state: UniV2DepositState | null
+  dispatch: React.Dispatch<UniV2DepositActions> | null
+}
+
+export const DepositContext = createContext<IDepositContext>({ state: null, dispatch: null })
```

Comment:
this is something we may want to consider zustand for in the future, just thinking out loud

Reply:
Oh I would be very happy to move this to Zustand, once we consolidate those DeFi view layer modules